docs/pages/blog/lab-date-pickers-to-mui-x.md

```diff
@@ -14,13 +14,17 @@ This means we'll be dedicating even more time and effort to these complex compon
 
 Date and time pickers are interface controls that enable the user to select a date (or time) from a menu.
 
+<img src="/static/blog/lab-date-pickers-to-mui-x/date-time-picker.png" style="width: 796px; margin-top: 16px; margin-bottom: 16px;" alt="Date and time picker component" />
```

Comment:
We will need to fix the background color at some points 😁. It should be

<img width="357" alt="Screenshot 2022-04-07 at 13 21 01" src="https://user-images.githubusercontent.com/3165635/162187719-14c56c23-2dd3-49ab-b5a4-1000727132b0.png">

not

<img width="343" alt="Screenshot 2022-04-07 at 13 21 05" src="https://user-images.githubusercontent.com/3165635/162187660-18e3b884-e1b3-44fc-9b49-da1de3077815.png">

right?

Reply:
Absolutely 😅 Thanks for pointing it out! Will resolve it here but we definitely should tackle it in a separate PR!